10.0/apache/docker-entrypoint.sh

```diff
@@ -18,7 +18,18 @@ if version_greater "$installed_version" "$image_version"; then
 fi
 
 if version_greater "$image_version" "$installed_version"; then
-    tar cf - --one-file-system -C /usr/src/nextcloud . | tar xf -
+    rsync -a --delete --exclude /config/ --exclude /data/ --exclude /apps/ /usr/src/nextcloud/ /var/www/html/
```

Comment:
Is there any reason why the tarball isn't directly extracted to /var/www/html in the Dockerfile? Then you wouldn't need to copy here.